bullet_train/docs/upgrades.md

```diff
@@ -28,7 +28,7 @@ git clean -d -f
 
 ```
 git fetch starter-repo
-````
+```
```

Comment:
Should we add `shell` to the beginning of the fenced blocks in this file?

Reply:
There are many such blocks. The "issue" with shell blocks is that there's almost no syntax highlighting in 90% of cases. I wanted to "build the MVP" of syntax highlighting functionality and have any/minimal discussion points left out and leave it to be added later. We already have the question of "should we use PRE or not" question, but that one is required to have this functionality.

Reply:
Fair enough, @vfonic. We'll leave these alone for now and can always add them later if we want them.